### DISCUSSION TITLE
Rendering placeholder instead of whitespace

### DISCUSSION THREAD

**Discussion by Bayramito** [Shopify/flash-list#760] [General]
How can i render kind of skeleton placeholder items instead of whitespaces ?

 Becouse my heights are atlest half of my device's height and if i scroll very fast i still can see some white spaces. 

And i want to replace these white spaces with my own component

**Comment by bloggerklik** [comment]
@Bayramito hi, did you find a solution for this idea?

**Reply by Bayramito** [reply]
Not yet

**Reply by bloggerklik** [reply]
@Bayramito thanks

**Comment by bloggerklik** [comment]
@Bayramito Hi, when I used slice() for data array performance improved and white space interestingly disappeared. I do not know the reason. Can you try it too? 

Example:
```
    const data = myData.slice(0);

    <FlashList
        data={data}
```

**Reply by Bayramito** [reply]
this doesnt work if you have large data  and complex UI for each item
